builtin/profiler/reporter.lua

```diff
@@ -73,105 +84,131 @@ local Formatter = {
 	end
 }
 
-local widths = { 55, 9, 9, 9, 5, 5, 5 }
-local txt_row_format = sprintf(" %%-%ds | %%%ds | %%%ds | %%%ds | %%%ds | %%%ds | %%%ds", unpack(widths))
+local max_fcol_width = 60
+local TxtFormatter = Formatter:new {
+	update = function(self)
+		local widths = self.widths
+		if not widths then
+			widths = { 30, 5, 5, 9, 9, 9 }
```

Comment:
1) Needs clarifying what these magic values do - and what they're used for (`-- Specifies each column width of the table output in characters`)
2) Shorter:
```Lua
self.widths = self.widths or { 30, 5, 5, 9, 9, 9 }
local widths = self.widths
```
Is localizing the variable needed here? I don't think that it's called so often that these microseconds actually bring a benefit.

Reply:
Not performance critical code, so not needed; just a habit. :-)

I'd still keep it localized here for the loop over `ins_stats` (grows with active instruments)
and having seen someone run the txt dump every few serverstep; it probably won't hurt.